### PR TITLE
docs: document how to test a custom exporter

### LINKED ISSUE #620: Document example on how to test a custom exporter
**Description**

Add documentation on how to test your custom exporter using this project.

### PR DESCRIPTION
**Pull request by npepinpe**
## Description

This PR adds documentation on how to test a custom exporter using this project.

## Related issues

closes #620 

## Pull Request Checklist

- [ ] All commit messages match our [commit message guidelines](https://github.com/camunda-cloud/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
- [ ] The submitting code follows our [code style](https://github.com/camunda-cloud/zeebe/wiki/Code-Style)
- [ ] If submitting code, please run `mvn clean install -DskipTests` locally before committing
- [ ] Ensure all PR checks are green

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -374,6 +374,185 @@ A series of examples are included as part of the tests, see
...
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@Testcontainers
+public final class MyExporterIT {
+  private static final DockerImageName TEST_IMAGE = DockerImageName.parse("ghcr.io/acme/exporter");
+
+  @Container
+  private final ZeebeContainer zeebe = new ZeebeContainer(TEST_IMAGE)
```
Comment (Zelldon) [inline README.md:544]: ❓ But this only works if this image builts on top of the zeebe image right ? 🤔

Comment (npepinpe) [reply README.md:544]: Yes, good point! It's what these projects do, i'll update the docs

**Review by Zelldon — APPROVED** [review]
🚀

**Review by nicpuppa — APPROVED** [review]
🍪

**Pull request merged**